frontend/src/app/components/containers/complaints/outcomes/oucome-by-animal/animal-outcome.tsx

```diff
@@ -122,179 +124,187 @@ export const AnimalOutcome: FC<props> = ({ index, data, agency, edit, remove })
   };
 
   return (
-    <div className="comp-animal-outcome">
-      <div className="equipment-item">
-        <div className="equipment-item-header">
-          <div className="title">
-            <h6>Animal {pad(animalNumber.toString(), 2)}</h6>
-          </div>
-          <div>
-            <CompTextIconButton
-              buttonClasses="button-text"
-              style={{ marginRight: "15px" }}
-              text="Delete"
-              icon={BsTrash3}
-              click={() => handleDeleteItem()}
-            />
-            <CompTextIconButton
-              buttonClasses="button-text"
-              text="Edit"
-              icon={BsPencil}
-              click={() => hendleEnableEditMode()}
-            />
+    <>
+      {showSectionErrors && (
+        <div className="section-error-message">
+          <BsExclamationCircleFill />
+          <span>Complete or delete section before closing the complaint.</span>
```

Comment:
There seems to be a bug in this section.  Steps to reproduce:

1) Enter an Animal with just species
2) Attempt to close ticket
3) Validation message is received to complete section
4) Fill in the rest of the fields and press save
5) Section is not saved.   Validation message persists.

Reply:
@afwilcox I was highlighted this bug in outcome animal in the demo, the bug is existing in the test env, not because of this PR. So even though the date in the outcome is showing up, you have actually click on it to make it take into affect. I have raised the bug ticket for it CE-800 and noticed Adrienne about it yesterday.

Reply:
@Scarlett-Truong - Okay thanks, I'll update the bug description and we can fix on a different PR